src/wallet/rpcwallet.cpp

```diff
@@ -2397,6 +2399,38 @@ UniValue lockunspent(const JSONRPCRequest& request)
 
         COutPoint outpt(uint256S(txid), nOutput);
 
+        const bool is_locked = pwallet->IsLockedCoin(outpt.hash, outpt.n);
+
+        if (fUnlock && !is_locked)
```

Comment:
Missing `{}` and `{` is one the same line. Same below. See [developer notes](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md).